src/relay/op/tensor/transform.cc

```diff
@@ -2069,12 +2070,11 @@ bool StridedSliceRel(const Array<Type>& types, int num_inputs, const Attrs& attr
       oshape[i] = tir::make_const(dshape[i].dtype(), (slice_range + step - 1) / step);
     }
   } else {
-    for (int64_t i = 0; i < num_axis; ++i) {
-      oshape[i] = Any();
-    }
+    CHECK(param->begin) << "strided_slice recieved invalid begin";
```

Comment:
print the received begin value in the message?

Reply:
If this check fails it will be a nullptr. Should I print the null?